Flickr/Flickr/ModelPhotoData.swift

```diff
@@ -115,7 +115,7 @@ class Photo {
                 return (url, scaleFactor)
             }
         }
-        scaleFactor -= 1
+        scaleFactor -= 0.999
```

Comment:
is this weird logic at least working? if yes it need to be extracted as it magic number right now